common-healing.lic

```diff
@@ -330,8 +330,8 @@ module DRCH
     waitrt?
     case result
     when *tend_dislodge
-      DRCI.dispose_trash(DRC.left_hand) if DRC.left_hand != snap.first
-      DRCI.dispose_trash(DRC.right_hand) if DRC.right_hand != snap.last
+      DRCI.dispose_trash(DRC.left_hand, get_settings.worn_trashcan, get_settings.worn_trashcan_verb) if DRC.left_hand != snap.first
```

Comment:
I'm not a fan of calling `get_settings` four times, or even calling it at all inside this method (it should be provided by the calling script).

That said, it won't hold up merging, it would just be nice to have it passed in (which I realize is a bigger change).